app/src/main/java/org/simple/clinic/scanid/IndiaNHIDInfo.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.scanid
+
+data class IndiaNHIDInfo(
```

Comment:
Why did we rename `PatientPrefillInfo`? It's fine. We just had to rename the payload.